newsfragments/46944.significant.rst

```diff
@@ -0,0 +1,16 @@
+Statically-coded executors are now deprecated
+
+The Statically-coded executors ``LocalKubernetesExecutor`` and ``CeleryKubernetesExecutor`` are now deprecated in favor of Multiple Executor Configuration. They will be removed in Airflow 3.0
+
```

Comment:
Technically there are no breaking changes, in this PR, but I feel like it's good to call it out in release notes that it's coming. Let me know what others think.

Reply:
I think this is fair.